Manifests/ManagedPreferencesApplications/com.twocanoes.xcreds.plist

```diff
@@ -489,7 +489,7 @@ A profile can consist of payloads with different version numbers. For example, c
 			<key>pfm_name</key>
 			<string>idpHostName</string>
 			<key>pfm_title</key>
-			<string>IDP Host Name</string>
+			<string>idpHostName</string>
```

Comment:
Did you mean to change the title to match the key name? Remember, `pfm_title` is the user-friendly name of the property.